Assets/Developers/Lab/Systems/MovementSystem.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Unity.Entities;
+using Unity.Mathematics;
+using Unity.Transforms;
+using UnityEngine;
+
+using Random = Unity.Mathematics.Random;
+
+namespace JanusVagabond
+{
+  public class MovementSystem : ComponentSystem
```

Comment:
It's a little bit unclear what MovementSystem really does. It seems that it affects multiple different entities with different types of components. I would add a comment to make it more clear what it does :^)

Reply:
Sounds good.

To explain why it affects multiple type of component, note that a system is freely to query any type of entity. I.e, MovementSystem can query entity from DayNightComponent. We just organize it like this at the moment for simple logic, but usually a System can query anything in the environment to implement its logic, as long as the logic is complete.

I.e, DayNight can query for monster spawner to spawn monster at night for example.